Contributing/readme.md

```diff
@@ -89,17 +93,29 @@ We also prefer to add a new line between HTML sibling elements to help visually
 </SomeComponent>
 ```
 
-## Assets
+### React Components
+Initially when rapidly developing [Phoenix](https://github.com/DoSomething/phoenix-next), we made some decisions on code file locations/hierarchy. Overtime we've come to refine this structure a bit, and are now working to update the Phoenix codebase with the latest decisions.
 
-### SVG
+We want to update our code directory structure within the `/components` directory to follow this pattern: 
 
-When adding a SVG to the codebase, be sure to run it SVGs through this optimizer: https://jakearchibald.github.io/svgomg.
+```
+- /components
+  |_ /actions
```

Comment:
Can we add `/artifacts` after `/actions`? It's a directory that made it into the system after I created the original draft of these updates in the discussion thread post:

```suggestion
  |_ /actions
  |_ /artifacts
```